docs/feature-support-matrix.md

```diff
@@ -27,7 +27,7 @@ An example:
 | ENS                      |         | Yes         | Yes          | No                | No                   | No               |
 | File data sources: IPFS  |         | Yes         | Yes          | No                | Yes                  | Yes              |
 
-The accepted `graph-node` version range is also specificied, with an "upgrade window" from the previous version.
+The accepted `graph-node` version range is also specificied, with an "upgrade window" from the previous version. Here's an example:
```

Comment:
I'm making it more clear that the `graph-node` version range here is just an example and not prescriptive, i.e. it shouldn't be updated for every new version.